src/Hungary/Mews.Fiscalizations.Hungary/Utils/Client.cs

```diff
@@ -30,7 +32,9 @@ internal Client(HttpClient httpClient, NavEnvironment environment)
         private async Task<HttpResponseMessage> SendRequestAsync<TRequest>(string endpoint, TRequest request)
             where TRequest : class
         {
-            var content = new StringContent(XmlManipulator.Serialize(request), Encoding.UTF8, "application/xml");
+            var data = new XmlSerializationData(namespaces: new XmlNamespace(ServiceInfo.XmlNamespace).ToEnumerable());
+            var xml = XmlSerializer.Serialize(request, data);
+            var content = new StringContent(xml.OuterXml, Encoding.UTF8, "application/xml");
```

Comment:
The encoding appears as a constant on a couple of places. Would it make sense to move it to ServiceInfo?